yarn.lock

```diff
@@ -485,10 +485,10 @@
   dependencies:
     tslib "^1.9.0"
 
-"@hmcts/rpx-xui-node-lib@^2.24.3":
-  version "2.24.3"
-  resolved "https://registry.yarnpkg.com/@hmcts/rpx-xui-node-lib/-/rpx-xui-node-lib-2.24.3.tgz#63292f848fb10a475d7aa65abeefa0b030a5ed9f"
-  integrity sha512-5pmLQIdJ3CdpxzMa+6IGFQ4+MB24yZ9Ocim7DlLvTe2jHQYZwE/TlbNGkhRjXmqPQy4bt/9zuk4X8IhyDgxtdA==
+"@hmcts/rpx-xui-node-lib@^2.24.5":
+  version "2.25.0"
```

Comment:
Is this the version we want? You seem to have specifically changed it in `package.json` to `^2.24.5` from `^2.24.3` but it's actually installed `2.25.0`. This looks like an unintentional change.

Reply:
This is intentional change

Reply:
So why specify `2.24.5` in the package.json? If you _want_ `2.25.0`, why not specify that?

Reply:
I get your question now. I tried it explicitly changing in yarn.lock. But, it moves it back to `"@hmcts/rpx-xui-node-lib@^2.24.5"` Its autogenerated by running yarn.

Reply:
Yeah, you can't change it in `yarn.lock`. It's happening when you do `yarn [install]` because it's set to `"^2.24.5"` in the `package.json`. The caret prefix ("^") indicates that it should use the latest compatible version so it'll get any newer minor or patch releases.

That _may_ ultimately be what we want to do but I'm guessing you chose `2.24.5` because you made changes in that version that you want to be part of this release. And you _didn't_ make changes in any versions from `2.24.6` onwards (or, if you did, they'll be in a different PR)...?

The safest way to test those changes in isolation is to remove the caret prefix in the `package.json` file and that's what I'd recommend.

Reply:
Thoughts on this, @Mo-Lala84?

Reply:
This is resolved now